device-types/Cisco/NCS-55A2-MOD-SE-S.yaml

```diff
@@ -0,0 +1,108 @@
+---
+manufacturer: Cisco
+model: NCS 55A2-MOD-SE-S
+slug: cisco-ncs-55a2-mod-se-s
+part_number: NCS-55A2-MOD-SE-S
+is_full_depth: false
+u_height: 2
+weight: 20.5
+weight_unit: lb
+module-bays:
+  - name: 0/0/1
+    label: MPA
+    position: 0/0/1
+  - name: 0/0/2
+    label: MPA
+    position: 0/0/2
+interfaces:
+  - name: MgmtEth0/RP0/CPU0/0
+    type: 1000base-t
+    mgmt_only: true
+  - name: TenGigE0/0/0/0
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/1
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/2
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/3
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/4
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/5
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/6
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/7
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/8
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/9
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/10
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/11
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/12
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/13
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/14
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/15
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/16
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/17
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/18
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/19
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/20
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/21
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/22
+    type: 10gbase-x-sfpp
+  - name: TenGigE0/0/0/23
+    type: 10gbase-x-sfpp
+  - name: TwentyFiveGigE0/0/0/24
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/25
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/26
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/27
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/28
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/29
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/30
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/31
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/32
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/33
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/34
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/35
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/36
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/37
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/38
+    type: 25gbase-x-sfp28
+  - name: TwentyFiveGigE0/0/0/39
+    type: 25gbase-x-sfp28
+power-ports:
+  - name: PSU0
```

Comment:
It appears that these are modular power supplies. Is this true?

If so, then the power supplies should be modelled as `module-bays` and the applicable PSUs should be added in the module-types folder if they do not already exist

Reply:
Power supplies are hot-swappable and 4 models are existing : 

NC55-1200W-ACFW | NCS 5500 AC 1200W Power Supply Port-S Intake / Front-to-back
NC55-930W-DCFW | NCS 5500 DC 930W Power Supply Port-S Intake / Front-to-back
NC55-900W-ACFW-HD | NCS 5500 AC 900W Power Supply Port-S Intake / Front-to-back
NC55-900W-DCFW-HD | NCS 5500 DC 900W Power Supply Port-S Intake / Front-to-back


Should we use device-bays instead of module-bays for power supplies ?

Reply:
These should be module-bays and each one of those devices should be added to the module-types folder